draft-ietf-quic-multipath.md

```diff
@@ -249,20 +249,22 @@ the use of the multipath extension during the connection handshake,
 as specified in {{QUIC-TRANSPORT}}. The new transport parameter is
 defined as follows:
 
-- initial_max_path_id (current version uses 0x0f739bbc1b666d09): the
-  initial_max_path_id transport parameter is included if the endpoint supports
-  the multipath extension as defined in this document. This is
-  a variable-length integer specifying the maximum path identifier
+- initial_max_path_id (current version uses 0x0f739bbc1b666d09): a
+  variable-length integer specifying the maximum path identifier
   an endpoint is willing to maintain at connection initiation.
-  For example, if initial_max_path_id is set to 1, only connection IDs
-  associated with Path IDs 0 and 1 should be issued by the peer.
-  If an endpoint receives an initial_max_path_id transport parameter with value 0,
-  the peer aims to  enable the multipath extension without allowing extra paths immediately.
-  To allow for the use of more paths later,
-  endpoints can send the MAX_PATH_ID frame to increase the maximum allowed path identifier.
+  This value MUST NOT exceed 2^32-1, as Path IDs are defined
+  with a maximum value 2^32-1 as the 32 bits of the Path ID
+  are used to calculate the nonce (see {{multipath-aead}}).
```

Comment:
Note, this is a normative change to explicitly state the MUST here. This is the same text as used for the MAX_PATHS frame value but I think we have to specify it here as well.

Reply:
How about this (referencing the nonce section directly rather than the intro but not repeating the value multiple times)?
```suggestion
  This value MUST NOT exceed 2^32-1, the maximum allowed value for the Path ID due to
  restrictions on the nonce calculation (see {{multipath-aead}}).
```

Reply:
Yes, that's good too.